data/ecosystems/h/hedera.toml

```diff
@@ -1,98 +1,31 @@
 # Ecosystem Level Information
 title = "Hedera"
 
-sub_ecosystems = ["Calaxy"]
-
-github_organizations = ["https://github.com/hashgraph"]
+sub_ecosystems = [
+"Calaxy", 
+"Heliswap", 
+"Hashport",
+"SaucerSwap Labs",
+"Heliswap",
+"Buidler Labs",
+"Blade Labs",
+"Burstall",
+"PieFi Platform",
+"Mintmaster Labs",
+"HashGuild",
+"TxMQ, Inc",
+"HSuite",
+"Trust Enterprises",
+"DOVU",
+]
+
+github_organizations = [
+"https://github.com/hashgraph",
+"https://github.com/limechain",
+"https://github.com/bugbytesinc/",
```

Comment:
This is a github user, not an org.  Are you able to add hedera repos they have created to this file directly?

Reply:
Yes will do!